### ISSUE TITLE
Add a GUI to work in parallel with command line version

### ISSUE THREAD

**Issue by nflorin** [Center-for-Research-Libraries/crl-serials-validator#37]
Build a (Tkinter) GUI to run in parallel with the current terminal version.

The Validator works perfectly well as is, so this is a lower priority.

**Comment by nflorin** [comment]
Putting this on hold until I've gotten the back end streamlined. It's not worth thinking about, especially as the command line version has gotten much better looking.
